BParkingNano/test/run_nano_cfg.py

```diff
@@ -282,8 +282,8 @@
         SelectEvents = cms.vstring(
                                    'nanoAOD_KMuMu_step', 
                                    'nanoAOD_Kee_step',
-                                   'nanoAOD_KstarMuMu_step',
-                                   'nanoAOD_KstarEE_step',
+                                   #'nanoAOD_KstarMuMu_step',
+                                   #'nanoAOD_KstarEE_step',
                                    )
 )
 
```

Comment:
Hi @ottolau, 
can you remove the changes to the run_nano_cfg.py ?
This is not part of your PR, will confuse the history and introduce in the master branch unwanted changes in general